src/main/java/net/sf/jabref/logic/importer/fetcher/MrDLibFetcher.java

```diff
@@ -92,6 +99,7 @@ private String makeServerRequest(String query) {
                 response = Unirest.get(query).asString().getBody();
             } catch (UnirestException e) {
                 LOGGER.error(e.getMessage(), e);
+                return "could not get a response.";
```

Comment:
When thinking longer, this should throw a `FetcherException`. Please quickly check the code of the other fetchers how they are doing localization.